mmv1/third_party/terraform/acctest/vcr_utils_test.go

```diff
@@ -449,3 +451,112 @@ func TestReformConfigWithProvider(t *testing.T) {
 		})
 	}
 }
+
+func TestDiffTestStepInjection(t *testing.T) {
```

Comment:
I couldn't find a reference, but I believe it is a Go convention to name tests `Test[FuncName]`, so I think this should be `TestInsertDiffSteps`